index.html

```diff
@@ -0,0 +1,137 @@
+      <div class="home_section_title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1>FOR THOSE WHO NEED TECHNOLOGY TO REVEAL THE BEAUTY OF PRODUCTS</h1>
+      </div>
+      <article>
+        <h4>Cloudflare Studio is a boutique design and software development company. We have 50+ JS and design experts on board and 6+ years of experience in building digital projects from scratch or refreshing existing ones.</h4>
+      </article>
+      <div>
+        <div class="info">
+          <h3>FROM (MOND 8AM) ~ (SUN 17AM)</h3>
+          <p>@ Full-Cycle product Development</p>
+        </div>
+      </div>
+    </section>
+
+    <section id="Program" class="program_section">
+        <h2>Why Choose Us?</h2>
+        <ul class="program">
+            <li class="program_1 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-1.svg" alt="agile">
+              </div>
+              <h3>Agile team </h3>
+              <p>An experienced team dedicated to only your project. There is no micromanagement. </p>
+            </li>
+            <li class="program_2 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-2.svg" alt="focus">
+              </div>
+              <h3>Focus on JS</h3>
+              <p>You will deal with a team of seasoned professionals who have been programing together for many years and understand each other. </p>
+            </li>
+            <li class="program_3 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-3.svg" alt="quality">
+              </div>
+              <h3>B.Quality designers</h3>
+              <p>As we started from design many years ago, we rank on Dribbble as 1 of the 10 best design studios around the world. </p>
+            </li>
+            <li class="program_4 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-4.svg" alt="creation">
+              </div>
+              <h3>Complex approach</h3>
+              <p>Our team brings together designers, software developers, QA engineers, and illustrators. </p>
+            </li>
+            <li class="program_5 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-8.svg" alt="time">
+              </div>
+              <h3>Short timeframes</h3>
+              <p>Our developers are featured as the best by TopCoder. Since 2013, we have already ticketed over 50,000 person-hours.</p>
+            </li>
+        </ul>
+        <div class="btn-program">
+          <button type="button">Join Fireart comminity </button>
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class="">
+          
+        </div>
+    </section>
```

Comment:
- [ ] For this section the background is looking okay but you can improve it to look like the design check images down:

Design
![og](https://user-images.githubusercontent.com/44745855/132837558-b3e87012-6b81-43f7-9b74-dd6a88f8adab.png)

your page
![main](https://user-images.githubusercontent.com/44745855/132837607-1e2bbcd0-fc6f-4a01-ad54-622d6efd69c2.png)

Reply:
Refer to that but anything else looks dope.

Reply:
Thanks